Symfony/CS/Fixer/Symfony/ExtraEmptyLinesFixer.php

```diff
@@ -16,10 +16,70 @@
 
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ * @author SpacePossum <possumfromspace@gmail.com>
  */
 final class ExtraEmptyLinesFixer extends AbstractFixer
 {
     /**
+     * @var array $cases {
+     *     @var int    Kind of Token.
+     *     @var string Method to call when token of kind is found.
+     * }
```

Comment:
this is wrong PSR-5 doc.
this syntax is reserved for hashes, see https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc.md#7-describing-hashes